src/js/global/main.js

```diff
@@ -159,6 +159,43 @@ $(document).ready(function(){
 		$('#do_beta').trigger('change');
 	}
 
+	/***
+	* Show popin upgrade
+	***/
+
+	var $wprUpgradePopin = $('.wpr-Popin-Upgrade'),
+	$wprUpgradeClosePopin = $('.wpr-Popin-Upgrade-close'),
+	$wprUpgradeOpenPopin = $('#wpr-popin-upgrade-toggle');
+
+	$wprUpgradeOpenPopin.click(function(e) {
+		e.preventDefault();
+		wprOpenUpgradePopin();
+		return false;
+	});
+
+	$wprUpgradeClosePopin.click(function() {
+		wprCloseUpgradePopin();
+		return false;
+	});
+
+	function wprOpenUpgradePopin(){
+		var vTL = new TimelineLite()
+			.set($wprUpgradePopin, {'display':'block'})
+			.set($wprPopinOverlay, {'display':'block'})
+			.fromTo($wprPopinOverlay, 0.6, {autoAlpha:0},{autoAlpha:1, ease:Power4.easeOut})
+			.fromTo($wprUpgradePopin, 0.6, {autoAlpha:0, marginTop: -24}, {autoAlpha:1, marginTop:0, ease:Power4.easeOut}, '=-.5')
+		;
```

Comment:
```suggestion
		var vTL = new TimelineLite();
		vTL
			.set($wprUpgradePopin, {'display':'block'})
			.set($wprPopinOverlay, {'display':'block'})
			.fromTo($wprPopinOverlay, 0.6, {autoAlpha:0},{autoAlpha:1, ease:Power4.easeOut})
			.fromTo($wprUpgradePopin, 0.6, {autoAlpha:0, marginTop: -24}, {autoAlpha:1, marginTop:0, ease:Power4.easeOut}, '=-.5')
		;
```

Reply:
Here the variable is unused. A slight change to first initialize the var and then use it for the chained methods.